tensornetwork/backends/jax/jax_backend.py

```diff
@@ -39,9 +39,12 @@ def convert_to_tensor(self, tensor: Tensor) -> Tensor:
     result = self.jax.jit(lambda x: x)(tensor)
     return result
 
-  def concat(self, values: Tensor, axis: int) -> Tensor:
+  def shape_concat(self, values: Tensor, axis: int) -> Tensor:
```

Comment:
Doubly come to think of it, we don't need the axis number for `shape_concat` at all since we always use just -1. Can you remove this argument?